app/controllers/posts_controller.rb

```diff
@@ -4,6 +4,7 @@ class PostsController < ApplicationController
 
   def index
     @posts = Post.order(id: :desc)
+    @posts = Post.page(params[:page]).per(7)
```

Comment:
マジックナンバーは避けておいた方がよいですね。
一番上に

```
PER_PAGE = 7
```

を定義して

```suggestion
    @posts = Post.page(params[:page]).per(PER_PAGE)
```

のようにするとよいですよ。